src/components/core/Header.vue

```diff
@@ -24,83 +43,279 @@ export default {
 </script>
 
 <template lang="pug">
-q-toolbar.row.q-py-sm.q-px-md.bg-grey-1
+q-toolbar.row.q-py-sm.q-px-md.bg-grey-1.gt-md
   router-link.row.items-center.cursor-pointer(:to='{ name: "home" }')
     img.logo.q-mr-md(src='~assets/legalcoin-full.png')
 
   .col.row.justify-center
-    search-bar.toolbar-select.q-mr-md
+    search-bar.toolbar-select.q-mr-md(size='100', v-if='$q.screen.gt.md')
 
   .q-mr-md.q-gutter-x-md.row.items-center.no-wrap(v-if='$q.screen.gt.sm')
-    router-link.toolbar-link(:to='{ name: "buy" }')
-      | Buy
-    router-link.toolbar-link(:to='{ name: "gallery" }')
-      | Gallery
+    q-btn.btn-left(
+      flat,
+      icon='grid_view',
+      label='Discover',
+      dense,
+      :to='{ name: "buy" }',
+      :class='(color = primary)'
+    )
+    q-btn(
+      flat,
+      icon='account_circle',
+      label='Account',
+      v-if='!isLoggedIn',
+      dense,
+      :to='{ name: "login" }'
+    )
 
   .row.items-center.no-wrap
-    q-btn.q-mr-sm(
-      v-if='$q.screen.gt.sm && isLoggedIn',
+    div
+      q-btn(
+        flat,
+        round,
+        dense,
+        :to='{ name: "mobile-search" }',
+        v-if='$q.screen.lt.sm'
+      )
+        q-icon.material-icons-outlined(name='search')
+      q-btn(flat, round, dense, :to='{ name: "buy" }', v-if='$q.screen.lt.md')
+        q-icon.material-icons-outlined(name='grid_view')
+      q-btn.q-ml-xs(
+        flat,
+        round,
+        dense,
+        v-if='!isLoggedIn && $q.screen.lt.md',
+        :to='{ name: "login" }'
+      )
+        q-icon.material-icons-outlined(name='account_circle')
+    q-btn.gt-sm(
       dense,
       flat,
       round,
-      size='md',
-      icon='wallet',
-      color='grey-6',
-      :to='{ name: "wallet" }'
+      no-wrap,
+      icon='account_circle',
+      label='Account',
+      v-if='isLoggedIn'
     )
-    //- q-btn(v-if="isLoggedIn" @click="logout") Logout
-    //- q-btn(v-if="!isLoggedIn" to="login" flat) Login
-    //- q-btn(v-if="!isLoggedIn" to="register" outline) Register
-    div(v-if='$q.screen.lt.md')
-      q-btn(flat, round, v-if='!isLoggedIn', dense, :to='{ name: "login" }')
-        q-icon.material-icons-outlined(name='account_circle')
-    div(v-if='$q.screen.gt.sm')
-      q-btn(dense, outline, v-if='!isLoggedIn', :to='{ name: "login" }')
-        q-avatar.q-mr-sm(rounded, size='30px')
-          q-icon.material-icons-outlined(name='account_circle')
-        .q-mr-sm Sign In
-    q-btn(dense, flat, round, size='md', no-wrap, v-if='isLoggedIn')
-      // TODO Change avatar when logged in
-      q-avatar(rounded, size='30px')
-        q-icon.material-icons-outlined(name='account_circle', color='grey-6')
-      //- q-icon(name='arrow_drop_down' size='16px')
-      q-menu(auto-close, fit)
+      q-menu.menu-edit(auto-close, fit)
         q-list(dense)
+          q-item.menu-link(
```

Comment:
Please change this menu so that the q-items are the only clickable items. In other words, remove the q-btn's. Tip: You can use the avatar prop of the q-item-section in conjunction with a [q-avatar](https://quasar.dev/vue-components/avatar#introduction) to display the icons